### PR TITLE
Tweak: Add heading and descriptions to admin settings pages [ED-15322]

### PR DESCRIPTION
**Pull request by rami-elementor**

### CODE REVIEW

**Review by KingYes — DISMISSED** [review]

File: includes/settings/settings.php [hunk 1]
```diff
@@ -252,6 +252,13 @@ protected function create_tabs() {
 				'label' => esc_html__( 'General', 'elementor' ),
 				'sections' => [
 					'general' => [
+						'label' => esc_html__( 'General', 'elementor' ),
+						'callback' => function() {
+							printf(
+								'<p>%s</p><br><hr><br>',
```
Comment (davseve) [inline includes/settings/settings.php:258]:
```suggestion
								'<p>%s</p><br/><hr/><br/>',
```

File: includes/settings/settings.php [hunk 2]
```diff
@@ -322,6 +329,13 @@ protected function create_tabs() {
 				'label' => esc_html__( 'Advanced', 'elementor' ),
 				'sections' => [
 					'advanced' => [
+						'label' => esc_html__( 'Advanced', 'elementor' ),
+						'callback' => function() {
+							printf(
+								'<p>%s</p><br><hr><br>',
```
Comment (davseve) [inline includes/settings/settings.php:335]: Same as above

File: includes/settings/settings.php (hunk 1 above)
Comment (rami-elementor) [reply includes/settings/settings.php:258]: Not sure about this. This is an XHTML convention. HTML5 allows self-closing tags.

Read this reference: https://developer.mozilla.org/en-US/docs/Glossary/Void_element

> Self-closing tags (`<tag />`) do not exist in HTML.
>
> If a trailing `/` (slash) character is present in the start tag of an HTML element, HTML parsers ignore that slash character.

File: includes/settings/settings.php (hunk 2 above)
Comment (rami-elementor) [reply includes/settings/settings.php:335]: See my answer above.

**Review by arielk — APPROVED** [review]

**Pull request merged**